docs/content/posts/2023-03-09-release-1.17.0.md

```diff
@@ -0,0 +1,485 @@
+[FLIP-211](https://cwiki.apache.org/confluence/display/FLINK/FLIP-211%3A+Kerberos+delegation+token+framework)
+was implemented which improves Flink's interactions with Kerberos: It reduces
+the number of requests that are necessary to exchange delegation tokens in Flink.
+
+
+# Upgrade Notes
+
+The Flink community tries to ensure that upgrades are as seamless as can be. However, certain
+changes may require users to make adjustments to certain parts of the program
+when upgrading to version 1.17. Please refer to the release notes for a
+comprehensive list of adjustments to make and issues to check during the
+upgrading process.
+
+
+# List of Contributors
+
+The Apache Flink community would like to express gratitude to all the
+contributors who made this release possible: 
+
+Ahmed Hamdy, Aitozi, Aleksandr Pilipenko, Alexander Fedulov, Alexander Preuß,
+Anton Kalashnikov, Arvid Heise, Bo Cui, Brayno, Carlos Castro, ChangZhuo Chen
+(陳昌倬), Chen Qin, Chesnay Schepler, Clemens, ConradJam, Danny Cranmer, Dawid
+Wysakowicz, Dian Fu, Dong Lin, Dongjoon Hyun, Elphas Toringepi, Eric Xiao,
+Fabian Paul, Ferenc Csaky, Gabor Somogyi, Gen Luo, Gunnar Morling, Gyula Fora,
+Hangxiang Yu, Hong Liang Teoh, HuangXingBo, Jacky Lau, Jane Chan, Jark Wu,
+Jiale, Jin, Jing Ge, Jinzhong Li, Joao Boto, John Roesler, Jun He, JunRuiLee,
+Junrui Lee, Juntao Hu, Krzysztof Chmielewski, Leonard Xu, Licho, Lijie Wang,
+Mark Canlas, Martijn Visser, MartijnVisser, Martin Liu, Marton Balassi, Mason
+Chen, Matt, Matthias Pohl, Maximilian Michels, Mingliang Liu, Mulavar, Nico
+Kruber, Noah, Paul Lin, Peter Huang, Piotr Nowojski, Qing Lim, QingWei,
+Qingsheng Ren, Rakesh, Ran Tao, Robert Metzger, Roc Marshal, Roman Khachatryan,
+Ron, Rui Fan, Ryan Skraba, Salva Alcántara, Samrat, Samrat Deb, Samrat002,
+Sebastian Mattheis, Sergey Nuyanzin, Seth Saperstein, Shengkai, Shuiqiang Chen,
+Smirnov Alexander, Sriram Ganesh, Steven van Rossum, Tartarus0zm, Timo Walther,
+Venkata krishnan Sowrirajan, Wei Zhong, Weihua Hu, Weijie Guo, Xianxun Ye,
+Xintong Song, Yash Mayya, YasuoStudyJava, Yu Chen, Yubin Li, Yufan Sheng, Yun
+Gao, Yun Tang, Yuxin Tan, Zakelly, Zhanghao Chen, Zhenqiu Huang, Zhu Zhu,
+ZmmBigdata, bzhaoopenstack, chengshuo.cs, chenxujun, chenyuzhi, chenyuzhi459,
+chenzihao, dependabot[bot], fanrui, fengli, frankeshi, fredia, godfreyhe,
+gongzhongqiang, harker2015, hehuiyuan, hiscat, huangxingbo, hunter-cloud09,
+ifndef-SleePy, jeremyber-aws, jiangjiguang, jingge, kevin.cyj, kristoffSC, kurt,
+laughingman7743, libowen, lincoln lee, lincoln.lil, liujiangang, liujingmao,
+liuyongvs, liuzhuang2017, luoyuxia, mas-chen, moqimoqidea, muggleChen, noelo,
+ouyangwulin, ramkrish86, saikikun, sammieliu, shihong90, shuiqiangchen,
+snuyanzin, sunxia, sxnan, tison, todd5167, tonyzhu918, wangfeifan, wenbingshen,
+xuyang, yiksanchan, yunfengzhou-hub, yunhong, yuxia Luo, yuzelin, zhangjingcun,
+zhangmang, zhengyunhong.zyh, zhouli, zoucao, 沈嘉琦, 1996fanrui
```

Comment:
I think `1996fanrui` and `Rui Fan` is the same guy.

Reply:
They owned same mail <1996fanrui@gmail.com>, I'll use `Rui Fan`